examples/autotls/main.go

```diff
@@ -58,6 +58,9 @@ func main() {
 		// User-Agent to use during DNS-01 ACME challenge
 		p2pforge.WithUserAgent(userAgent),
 
+		// Optional extra delay before the initial registration
+		p2pforge.WithRegistrationDelay(10*time.Second),
```

Comment:
With this registration delay, do we still need the 5 second sleep below (to confirm our public IPs)? 

https://github.com/libp2p/go-libp2p/pull/3211/files#diff-b413b2c049e0ed402c4b3fe4f1501acded0f28dea3a64d709e88809be9b3a867R137